timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -108,6 +116,11 @@ class ChronosModel(AbstractTimeSeriesModel):
     default_num_samples: int = 20
     default_model_path = "autogluon/chronos-t5-small"
     maximum_context_length = 512
+    fine_tuned_ckpt_name: str = "fine-tuned-ckpt"
+    default_fine_tune_lr: float = 0.001
```

Comment:
This will be changed later based on experiments. For now I have found that 5K steps with 1e-4 initial rate is good default value. This may also change though.